src/common/actions/repositories.js

```diff
@@ -67,9 +68,15 @@ export function fetchUserRepositories(pageNumber) {
           if (result.pageLinks) {
             dispatch(setPageLinks(result.pageLinks));
           }
+        })
+        .catch(error => {
+          return Promise.reject(error);
         });
       })
-      .catch(error => dispatch(fetchRepositoriesError(error)));
+      .catch(error => {
+        log.warn(error);
```

Comment:
I've reworked the error handling here because errors thrown by actions inside the promise were being swallowed. The errors were actually being thrown inside a `src/common/components/select-repository-list/index.js`, so why this would be swallowed by this promise is not known.